metrics/local.md

```diff
@@ -0,0 +1,29 @@
+# local
+
+The `local` metrics plugin stores the values on memory.
+
+## Parameters
+
+### `default_labels`
+
+Specifies the default labels for the metrics. \(default: `{agent: "Fluentd", hostname: "#{Socket.gethostname}"}`\)
+
+### `labels`
+
+Specifies other custom labels for the metrics. \(default: `{}`\)
+
+## Example
+
+With this configuration:
+
+```text
+<system>
+  <metrics>
+    @type local
+  </metrics>
+</system>
+```
+
```

Comment:
It seems that Fluentd's user may be confused what can do with bult-in `@type local`? It is not explained well.
It says that "The `local` metrics plugin stores the values on memory." but how to access it?

It may be covered with fluent-plugin-metrics-cmetrics, but it may be unfriendly.

Reply:
No, built-in `local` type plugin should covered the previous usages for storing single numerical values.
This behavior will be changed by other 3rd party plugins but built-in plugin should have equivalent functionality before Fluentd v1.13 series do.

Reply:
And I think that this question should be solved by adding fluent-plugin-metrics-cmetrics' plugin descriptions.
This plugin has very interesting functionality, which is native prometheus/influxdb format support.
These native supported format will handle for more rich labeled prometheus metrics and fast format translations among cmetrics, influxdb, and prometheus' counter/gauge formats.
This interesting concepts and usage will be revealed within this month. 😛

Reply:
>  I think that this question should be solved by adding fluent-plugin-metrics-cmetrics' plugin descriptions.

If so, it seems ok.